webpack/assets/javascripts/hosts/tableCheckboxes.js

```diff
@@ -68,6 +73,15 @@ function toggleActions() {
 // setups checkbox values upon document load
 $(document).on('ContentLoad', () => {
   if (window.location.pathname !== foremanUrl('/hosts')) return;
+  const hostQuery = sessionStorage.getItem('hostQuery');
```

Comment:
any chance to add tests to these? it would be the 4th time this code breaks.. ;)